deployment/k8s/texera-helmchart/Chart.lock

```diff
@@ -0,0 +1,18 @@
+dependencies:
```

Comment:
I think this file should not be included.

Reply:
removed